src/config/map.d

```diff
@@ -316,7 +316,7 @@ public:
 		return format!"[%-(%s, %)]"(entries.map!(e => e.dump()));
 	}
 
-	hash_t toHash() const {
+	hash_t toHash() const nothrow @trusted {
```

Comment:
NB: the compiler wants `@safe`; wasn't an option here, so I've just slapped an `@trusted` here without further analysis.